src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,30 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards) to manage releases. Read
+“[Tracking progress on your project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards)"
```

Comment:
These two quotes are not the same style. I guess that's a non-blocker but I think it would be good to be consistent, at least consistent within the same quote.

```
# The second one, an ASCII double quote
$ echo "\"" | tr -d "\n" | od -An -t dC
   34
# The first one, I guess a unicode double quote?
$  echo "“" | tr -d "\n" | od -An -t dC
  -30 -128 -100
```

Reply:
ascii one makes more sense IMHO